iml_common/blockdevices/blockdevice_zfs.py

```
@@ -507,6 +507,12 @@ def initialise_driver(cls, managed_mode):
         except OSError:
             pass
 
+        # remove json store for zfs objects
+        try:
+            os.remove('/tmp/store.json')
+        except OSError:
+            pass
```

What if the OSError here is not ENOENT?  Is it really safe to just ignore any error trying to remove this file?